### ISSUE TITLE
[DISCUSS]root user-options granted_admin Parameter modification is ambiguous

### ISSUE THREAD

**Issue by Benxiaohai001** [cnosdb/cnosdb#1702]
**Describe the issue**
A clear and concise description of issue.
root user-options granted_admin Parameter modification is ambiguous
https://github.com/cnosdb/docs/blob/main/docs/zh/v2.4/manage/tenant.md#admin%E6%9D%83%E9%99%90
![image](https://github.com/cnosdb/cnosdb/assets/40594937/46e717a4-b1fc-4bf0-a687-15cff649da93)
![image](https://github.com/cnosdb/cnosdb/assets/40594937/67f6ec5b-3b9d-4a52-9753-c7927073519c)

Through the description in the document, and the actual operation. I understand that granted_admin indicates whether the permission is granted or not, and that the root user is the initial permission, so it should not be changed here.

**What you like us to do**
Please give us some advices

**Additional context**
Add any other context about the issue here.

**Comment by roseboy-liu** [comment]
I agree with your opinion.

**Comment by Benxiaohai001** [comment]
```log
public ❯ alter user root set granted_admin = false;
422 Unprocessable Entity, details: {"error_code":"010024","error_message":"Invalid parameter : The root user does not support changing granted_admin"}
```
version: cnosdb 2.4.0, revision db434d4696e0c58e6881cc89e49cabece0316bff
Modified according to the above suggestions;

**Comment by Benxiaohai001** [comment]
@roseboy-liu  The PR to fix this issue needs to be merged into 2.3.

**Comment by Benxiaohai001** [comment]
```log
public ❯ alter user root set granted_admin = true;
422 Unprocessable Entity, details: {"error_code":"010024","error_message":"Invalid parameter : The root user does not support changing granted_admin"}
```
version: cnosdb 2.3.4, revision 07e229d4081c61b5012f70c3254e534ffe05d2e9
